docs/neopixel.rst

```diff
@@ -51,11 +51,17 @@ Classes
 
 .. py:class::
     NeoPixel(pin, n)
+    NeoPixel(pin, n, 4)
 
     Initialise a new strip of ``n`` number of neopixel LEDs controlled via pin
-    ``pin``. Each pixel is addressed by a position (starting from 0). Neopixels
-    are given RGB (red, green, blue) values between 0-255 as a tuple. For
-    example, ``(255,255,255)`` is white.
+    ``pin``. For **V2** micro:bit boards that support RGBW neopixels, a third
+    value can be passed to ``NeoPixel`` to indicate the number of bytes per
```

Comment:
A third "argument"

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/694/commits/474f3cc7442f540d016ee946f14475666bcb78a6